react-stopwatch/src/App.js

```diff
@@ -3,6 +3,10 @@ import styl from './App.module.css';
 import StopwatchButtonsContainer from './Components/StopwatchButtons/StopwatchButtonsContainer.jsx';
 import StopwatchTimeContainer from './Components/StopwatchTime/StopwatchTimeContainer';
 
+
+const OUR_NEW_VARIABLE = 'LOL';
```

Comment:
use camelCase for variables : OUR_NEW_VARIABLE => ourNewVariable

Reply:
Fixed!